guide/popular-topics/embeds.md

```diff
@@ -74,7 +74,6 @@ const exampleEmbed = new MessageEmbed()
 		{ name: 'Inline field title', value: 'Some value here', inline: true },
 		{ name: 'Inline field title', value: 'Some value here', inline: true },
 	)
-	.addField('Inline field title', 'Some value here', true)
```

Comment:
This should be replaced with addFields example

Reply:
I can't seem to find this 🤔

Reply:
i mean instead of just removing the code we should have `.addFields({ name: 'Inline field title', value: 'Some value here', inline: true})`

Reply:
Oh alright